src/ussd_pmtct.js

```diff
@@ -3,17 +3,86 @@ go.app = function() {
     var App = vumigo.App;
     var Choice = vumigo.states.Choice;
     var ChoiceState = vumigo.states.ChoiceState;
+    var PaginatedChoiceState = vumigo.states.PaginatedChoiceState;
     var EndState = vumigo.states.EndState;
     var FreeText = vumigo.states.FreeText;
+    var JsonApi = vumigo.http.api.JsonApi;
+    var HttpApi = vumigo.http.api.HttpApi;
 
+    var IdentityStore = require('seed_jsbox_utils').IdentityStore;
+    var StageBasedMessaging = require('seed_jsbox_utils').StageBasedMessaging;
+    var utils = require('seed_jsbox_utils').utils;
 
     var GoNDOH = App.extend(function(self) {
         App.call(self, "state_start");
         var $ = self.$;
-        // var interrupt = true;
+        // var interrupt = true
+
+        // variables for services
+        var is;
+        var sbm;
 
         self.init = function() {
-            // init services
+            // initialising services
+            var base_url = self.im.config.services.identity_store.prefix;
+            var auth_token = self.im.config.services.identity_store.token;
+            is = new IdentityStore(new JsonApi(self.im, null), auth_token, base_url);
+
+            base_url = self.im.config.services.stage_based_messaging.prefix;
+            auth_token = self.im.config.services.stage_based_messaging.token;
+            sbm = new StageBasedMessaging(new JsonApi(self.im, null), auth_token, base_url);
+        };
+
+
```

Comment:
Please put a comment in here that indicates that the functions below are essentially temporary while we're still loading information from the old system.